### ISSUE TITLE
Official `typing_extensions 4.13.0` release begging :beaver: 

### ISSUE THREAD

**Issue by leycec** [python/typing_extensions#518]
@JelleZijlstra, @brianschubert, @Daraan, @AlexWaygood, @hauntsaninja: Merry almost-Christmas, wonderful fellow Typonistas. You may know me from such hybrid runtime-static type-checkers as @beartype. On behalf of @beartype and our upcoming `beartype 0.20.0` release, I shall now proceed to beg ineffectually for an official `typing_extensions 4.13.0` release.

@beartype internally depends on the unreleased PEP 747-compliant `typing.TypeForm[...]` type hint factory, because it seemed like a good idea at the time. It now seems like a bad idea and I regret my life choices. Using `TypeForm` is kinda complicated by the lack of a concomitant backport in `typing_extensions`... *or any support whatsoever from anyone anywhere.*

I've temporarily hacked around this omission via some kludgy `if typing.TYPE_CHECKING:` ugliness in the @beartype codebase, which absolutely doesn't pass inspection. I'm unconvinced my kludges will actually hold up to scrutiny by end users in the absence of a `typing_extensions` release officially backporting `TypeForm`. I'm shook.

The next stable release of @beartype is thus blocked by the next stable release of `typing_extensions`. This is my fault, of course. Let this be a lesson to never depend on anyone. I hate release begging, but I am now release begging. It's been literally six months since the last stable release of `typing_extensions`, which is almost as bad as the @beartype release schedule. Which is to say... it's bad. Things don't get worse than the @beartype release schedule. :rofl: 

## In Conclusion, May `typing_extensions` Endure Forever

Thanks so much for all the amazing `typing` backports over these many years, y'all! May Python win all the glory in 2025.

**Comment by JelleZijlstra** [comment]
Yes, we should release soon. Note that by our policy, we'll have at least a one-week wait between a release candidate and final release.

I still want to add PEP 728 support too but haven't gotten around to it yet.

**Comment by hauntsaninja** [comment]
Note that typeshed will also need a `TypeForm` in typing_extensions.pyi

**Comment by leycec** [comment]
> Note that by our policy, we'll have at least a one-week wait between a release candidate and final release.

Indeed, this is the @beartype release policy as well: **safety first.** :safety_vest: :goggles: 

Thanks for the prompt attention, y'all. I see that the future of QA is in excellent hands.

**Comment by brianschubert** [comment]
@leycec FYI, 4.13.0rc1 is now live: https://github.com/python/typing_extensions/releases/tag/4.13.0rc1
